src/api/eventTypes.js

```diff
@@ -158,6 +160,18 @@ export type UpdateMessageFlagsEvent = $ReadOnly<{|
   flag: empty, // TODO fill in
   all: boolean,
   messages: $ReadOnlyArray<number>,
+
+  // Added in server-5.0
+  message_details?: {|
+    [string]: {|
+      type: string,
+      mentioned: boolean,
```

Comment:
This can be updated with a recent improvement to the API:
```suggestion
      mentioned?: true,
```